diff --git a/dist/server/lib/start-server.js b/dist/server/lib/start-server.js
index d4cbc252101382891581422f56557f2aa62aa809..f10fd7d0070ec9a3f3807f85e83f4601cd219841 100644
--- a/dist/server/lib/start-server.js
+++ b/dist/server/lib/start-server.js
@@ -42,6 +42,7 @@ const _trace = require("../../trace");
 const _ispostpone = require("./router-utils/is-postpone");
 const _isipv6 = require("./is-ipv6");
 const _asynccallbackset = require("./async-callback-set");
+const _logger = require("pino-http")();
 function _interop_require_default(obj) {
     return obj && obj.__esModule ? obj : {
         default: obj
@@ -150,6 +151,7 @@ async function startServer(serverOptions) {
         });
     }
     async function requestListener(req, res) {
+        if (!/^(\/_next\/static|\/_next\/image|\/__nextjs_original-stack-frame|\/__nextjs_source-map|\/icon.svg)/.test(req.url)) _logger(req, res);
         try {
             if (handlersPromise) {
                 await handlersPromise;
