diff --git a/dist/server/lib/start-server.js b/dist/server/lib/start-server.js
index 05cfbd12afb07ad54a444dfcfd84863b28503781..e3c8bda2a6f33493293049ab17d4a2c20fd2859e 100644
--- a/dist/server/lib/start-server.js
+++ b/dist/server/lib/start-server.js
@@ -41,6 +41,7 @@ const _turbopackwarning = require("../../lib/turbopack-warning");
 const _trace = require("../../trace");
 const _isipv6 = require("./is-ipv6");
 const _asynccallbackset = require("./async-callback-set");
+const _logger = require("pino-http")();
 function _interop_require_default(obj) {
     return obj && obj.__esModule ? obj : {
         default: obj
@@ -149,6 +150,7 @@ async function startServer(serverOptions) {
         });
     }
     async function requestListener(req, res) {
+        if (!/^(\/_next\/static|\/_next\/image|\/__nextjs_original-stack-frame|\/__nextjs_source-map|\/icon.svg)/.test(req.url)) _logger(req, res);
         try {
             if (handlersPromise) {
                 await handlersPromise;
