diff --git a/dist/server/lib/start-server.js b/dist/server/lib/start-server.js
index 96db246d9419d0c1147e49b4a108b0dec8d362cd..8898f3f8ba7f4d491daad442b29d8cea985435a0 100644
--- a/dist/server/lib/start-server.js
+++ b/dist/server/lib/start-server.js
@@ -41,6 +41,7 @@ const _turbopackwarning = require("../../lib/turbopack-warning");
 const _trace = require("../../trace");
 const _isipv6 = require("./is-ipv6");
 const _asynccallbackset = require("./async-callback-set");
+const _logger = require("pino-http")();
 function _interop_require_default(obj) {
     return obj && obj.__esModule ? obj : {
         default: obj
@@ -149,6 +150,7 @@ async function startServer(serverOptions) {
         });
     }
     async function requestListener(req, res) {
+        if (!/^(\/_next\/static|\/_next\/image|\/__nextjs_original-stack-frame|\/__nextjs_source-map|\/icon.svg)/.test(req.url)) _logger(req, res);
         try {
             if (handlersPromise) {
                 await handlersPromise;
